bytesize.go

```diff
@@ -0,0 +1,135 @@
+package userconfig
```

Comment:
I like starting stuff in the place where it  used and if we then get the feeling that it is useful move it further along. No need to put everything immediately into generic-types-go and make it an uber-library.